reporter/src/main/kotlin/reporters/AmazonOssAttributionBuilderReporter.kt

```diff
@@ -108,10 +108,7 @@ class AmazonOssAttributionBuilderReporter : Reporter {
 
                 val allCopyrights = licenseInfo.filter(LicenseView.ONLY_DETECTED).flatMapTo(mutableSetOf()) {
                     it.getCopyrights(omitExcluded = true)
-                }.ifEmpty {
-                    // This cannot be empty.
-                    setOf("No Copyright detected.")
-                }
+                }.ifEmpty { setOf("No Copyright detected.") }
```

Comment:
I'm not 100% sure the comment is redundant here. Without it, maybe someone is tempted to change `.ifEmpty { setOf("No Copyright detected.") }` to `.orEmpty()` in there future, but that would not work, as the Amazon Attribution builder forbids the set of copyrights to be empty.

So that's what the comment intended to say: "We must use something non-empty here or else the Attribution builder's API will reject the request."

Reply:
Ok, that's a different story then. That intention was not obvious to me at all from the comment. Let me change the commit.

Reply:
I changed the comment instead.

Reply:
Actually, I liked your comment `// The copyright set must not be empty as otherwise the Attribution builder rejects the request.` better than mine as it uses passive language, which is what we commonly do.

Reply:
Done.